docs/components/concepts/process-instance-migration.md

```diff
@@ -141,6 +141,129 @@ The called process instance is not changed when migrating the call activity.
+This is especially useful when a subscription behaviour is changed in the new version of the process definition, but you want to keep the catch event behaviour of the running processes the same after the migration.
+In the following section we will discuss situations where mapping a catch event may or may not be useful.
+
+#### When to map a catch event?
+
+<!--
+- Definition of the catch event does not change
+  - You want to keep the event subscription the same
+  - You want to reset event subscription
+- Definition of the catch event changes
+  - You want to keep the old event subscription
+  - You want to use the new catch event definition to re-initiate the subscription
+-->
+
+Choosing to map the catch event or not gives you control over what happens to the event subscription.
+Let's explain both mapping and non-mapping scenarios with examples.
+
+#### A mapping instruction is provided between the catch events
+
+An active user task has been waiting for a timer boundary event.
+The timer boundary event is defined as a duration of one week.
+The user task has not been completed and has already spent five days waiting for the timer.
+
+![The process instance is waiting at the active user task A with a timer boundary event attached.](assets/process-instance-migration/migration-catch-event-source.png)
+
+Now we want to [change an inactive part of the process](#changing-the-process-instance-flow-for-inactive-parts) by adding a user task after the user task with the timer boundary event.
+When migrating the process instance, we want to keep the timer unchanged.
+Instead of waiting for the full time defined by the target process' timer boundary event, we only want to wait for the remaining two days.
+To achieve that, you must map the timer boundary event to the timer boundary event in the target process.
+This ensures the timer is migrated (_the associated subscription is migrated_) and the duration is preserved.
+To achieve this for the example above, the mapping between active user tasks `A` -> `A` and timer boundary events `Timer1` -> `Timer2` must be provided.
+Assuming that the timer boundary event is defined as 2 weeks duration in the target process, the process instance will look like following after the migration:
+
+![The process instance is waiting at the active user task A with the migrated timer boundary event attached.](assets/process-instance-migration/migration-catch-event-different-target.png)
+
+Mapping catch events applies to other event types as well.
+For example, if you want to keep the message name the same for a message event subprocess start event, you should map the message start event when migrating the process instance.
+Another example would be to preserve a signal name for an intermediate signal catch event, you should map the signal catch event when to the one in the target while migrating the process instance.
```

Comment:
This behavior is specific to event-based gateway. A regular intermediate catch event **must always be mapped** because it's an active element.

```suggestion
Another example would be to preserve a signal name for an intermediate signal catch event attached to an event-based gateway. In this case, you should map the signal catch event to the one in the target while migrating the process instance.
```